### PR TITLE
Focus on geometry for readers to understand easier

### PR DESCRIPTION
**Pull request by songyingxu**
Add a focus line.
```
Map.centerObject(geometry);
```

### CODE REVIEW

**Comment by tylere** [comment]
We appreciate that you are trying to contribute to a community tutorial. However, your GitHub profile currently features sexualized imagery that may be offensive to others. This is not consistent with Google's standards for fostering an open and welcoming environment, and therefore we will not review this pull request. 
 
For more information on Google's Community Guidelines see:
- https://opensource.google/conduct/
- https://opensource.google/docs/releasing/template/CODE_OF_CONDUCT/#our-standards

**Comment by songyingxu** [comment]
Yes,&nbsp;you are right. The number '3'&nbsp;is not necessary.
I tried using&nbsp;[Map.centerObject(geometry, 3);] and&nbsp;found that the polygon is too small, so [Map.centerObject(geometry);] will be OK!
&nbsp;







宋英旭



东华理工大学/教职工/教辅机构/网络与信息中心（信息化建设与管理办公室）

19100100196

13163337265

东华理工大学




&nbsp;
&nbsp;
&nbsp;
------------------&nbsp;原始邮件&nbsp;------------------
发件人: "Tyler Erickson"; 
发送时间: 2020年4月29日(星期三) 凌晨0:33
收件人: "google/earthengine-community"; 
抄送: "Yingxu Song"; "Author"; 
主题: Re: [google/earthengine-community] Focus on geometry for readers to understand easier (#165)

&nbsp;


 
@tylere requested changes on this pull request.
 
The proposed changes do not appear to work correctly. Did you mean to set the zoom level when centering the object?
 Map.centerObject(geometry, 3);

File: tutorials/beginners-cookbook/index.md [hunk 1]
```diff
@@ -818,7 +818,8 @@ print(coll);
 // Create rectangle over Dubai.
 var geometry = ee.Geometry.Rectangle([55.1, 25, 55.4, 25.4]);
 // Add layer to map.
-Map.addLayer(geometry);
+Map.addLayer(geometry, 3);
+Map.centerObject(geometry);
```
Comment (tylere) [inline tutorials/beginners-cookbook/index.md]:
```suggestion
Map.addLayer(geometry);
Map.centerObject(geometry, 3);
```

**Comment by gino-m** [comment]
Closing due to inactivity.

**Pull request closed without merging**